src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser.cs

```diff
@@ -59,10 +60,28 @@ public static NamingStylePreferences ParseDictionary(IReadOnlyDictionary<string,
                 }
             }
 
-            return new NamingStylePreferences(
+            var preferences = new NamingStylePreferences(
```

Comment:
You may want to stick a comment here to clarify that the namingRules you pass *in* and the preferences.Rules that you get *out* are actually different things. Otherwise this looked like you were creating something just to not use it in any interesting way.

Reply:
I decided to leave this one for now.